_pages/welcome-to-TTS/getting-started.md

```diff
@@ -6,164 +6,164 @@ tags:
+to which you are invited. Listen and ask questions.
+Tell us if you think we're doing things wrong.
+Tell us about your pets. Just get in there and get actively involved.
 You can always say "no" later, but right now you can learn a lot by just saying yes.
 
-Remember, you don't need to have any specific expertise to join a guild or working group: 
-just join their Slack channel and start attending their standing meetings. 
+Remember, you don't need to have any specific expertise to join a guild or working group:
+just join their Slack channel and start attending their standing meetings.
 If you need an invite, post a message in the Slack channel or DM whoever organizes the group.
 
 ## Technical considerations
 
-TTS is a technical organization, so even if you're not an engineer you will find yourself in technical environments. 
+TTS is a technical organization, so even if you're not an engineer you will find yourself in technical environments.
 Here are the bits everyone will probably need to know:
 
 ### Github and Version control
-Version control is simply the act of keeping track of a file without having to name it 
-"myfile-10-10-18-final-final-final-approved." 
-Instead of creating a new version of a file for each change, you use a system to keep track of revisions. 
-This is especially important when you're talking about a whole bunch of files working together, like code, or a large multi-page document. 
-In that case, you really need a version control system. 
+Version control is simply the act of keeping track of a file without having to name it
+"myfile-10-10-18-final-final-final-approved."
+Instead of creating a new version of a file for each change, you use a system to keep track of revisions.
+This is especially important when you're talking about a whole bunch of files working together, like code, or a large multi-page document.
+In that case, you really need a version control system.
 Here at TTS, our version control system of choice is GitHub.
 
-Almost everyone at TTS spends some time working with GitHub, whether they're an engineer or not. 
-If this is new to you we suggest you start with our [Intro to Github]({{site.baseurl}}/intro-to-github/). 
-When you're ready to begin using GitHub, the [Handbook repository](https://github.com/18F/handbook) is a great place to practice basics like creating a branch and a pull request. 
-If you see anything in the Handbook that appears out of date or incorrect, you can practice your new pull request skills and help us improve it! 
+Almost everyone at TTS spends some time working with GitHub, whether they're an engineer or not.
+If this is new to you we suggest you start with our [Intro to Github]({{site.baseurl}}/intro-to-github/).
+When you're ready to begin using GitHub, the [Handbook repository](https://github.com/18F/handbook) is a great place to practice basics like creating a branch and a pull request.
```

Comment:
@afeld seems like `https://github.com/18F/handbook` should also be managed in config file too?

Reply:
🤷‍♂It's quite easy to find-and-replace that should we ever move the repository, but I would merge a pull request switching to a config variable.